src/lib/components/Badge.tsx

```diff
@@ -0,0 +1,14 @@
+import styled from 'lib/theme'
+
+import Row from './Row'
+
+export const Badge = styled(Row)<{ borderRadius?: number; padding?: string }>`
+  background-color: ${({ theme }) => theme.outline};
+  border-radius: ${({ borderRadius }) => `${borderRadius ?? 0.5}em`};
+  padding: ${({ padding }) => padding ?? '0.25em 0.375em'};
+  z-index: 2;
+`
+
+export const BadgeDark = styled(Badge)`
```

Comment:
Instead of a separate BadgeDark, could you include `color` as a prop for `Badge`, and default it to `outline`: `background-color: ${({ theme, color = 'outline' }) => theme[color]};`? That way you don't need a separate BadgeDark.